### PR TITLE
blocklogger: Show votes, tickets, & revocations.

### LINKED ISSUE #739: display votes / tickets in block processing log
instead of saying
Processed 1 block ... (25 transactions, 10 stake transactions, height 149676
do this
Processed 1 block ... (25 transactions, 5 votes, 5 tickets, height 149676

### PR DESCRIPTION
**Pull request by dnldd**
Fixes #739

### CODE REVIEW

File: blockmanager.go [hunk 1]
```diff
@@ -724,9 +725,17 @@ func (b *blockManager) logBlockHeight(block *dcrutil.Block) {
 	if b.receivedLogTx == 1 {
 		txStr = "transaction"
 	}
-	bmgrLog.Infof("Processed %d %s in the last %s (%d %s, height %d, %s)",
+	stakeStr := "tickets"
+	if stakeTxs == 1 {
+		txStr = "ticket"
```
Comment (dajohi) [inline blockmanager.go]: stakeStr

File: blockmanager.go [hunk 2]
```diff
@@ -724,9 +725,17 @@ func (b *blockManager) logBlockHeight(block *dcrutil.Block) {
...
+	}
+	voteStr := "votes"
+	if votes == 1 {
+		txStr = "vote"
```
Comment (dajohi) [inline blockmanager.go]: voteStr

Comment (dnldd) [inline blockmanager.go]: corrections made, pr updated.

**Comment by davecgh** [comment]
Please test changes before submitting a PR.  This doesn't work.

**Comment by davecgh** [comment]
This will need to be updated to deal with the merge conflicts introduced by #787 as far as the removal of the dead code.  I went ahead and merged that since it also removed the unused fields which this one doesn't do.

**Comment by davecgh** [comment]
Also, please update the commit title to match the PR title which includes the prefix with the system being modified.

File: blocklogger.go [hunk 3]
```diff
@@ -12,10 +12,13 @@ import (
 // to show users progress of certain "actions" involving some or all current
 // blocks. Ex: syncing to best chain, indexing all blocks, etc.
 type blockProgressLogger struct {
-	receivedLogBlocks int64
-	receivedLogTx     int64
-	receivedLogSTx    int64
-	lastBlockLogTime  time.Time
+	receivedLogBlocks      int64
+	receivedLogTx          int64
+	receivedLogSTx         int64
```
Comment (davecgh) [inline blocklogger.go]: This doesn't seem to be needed anymore since it's not displayed.

File: blocklogger.go [hunk 4]
```diff
@@ -41,9 +44,11 @@ func (b *blockProgressLogger) logBlockHeight(block *dcrutil.Block) {
 	b.Lock()
 	defer b.Unlock()
 	b.receivedLogBlocks++
-	b.receivedLogTx += int64(len(block.MsgBlock().Transactions))
 	b.receivedLogSTx += int64(len(block.MsgBlock().STransactions))
```
Comment (davecgh) [inline blocklogger.go]: No need for this anymore since it's not used.

File: blocklogger.go [hunk 5]
```diff
@@ -63,19 +68,31 @@ func (b *blockProgressLogger) logBlockHeight(block *dcrutil.Block) {
...
 		b.progressAction, b.receivedLogBlocks, blockStr, tDuration,
-		b.receivedLogTx, txStr, b.receivedLogSTx, stxStr, block.Height(),
-		block.MsgBlock().Header.Timestamp)
+		b.receivedLogTx, txStr, b.receivedLogVotes, voteStr,
+		b.receivedLogTickets, ticketStr, b.receivedLogRevocations,
+		revocationStr, block.Height(), block.MsgBlock().Header.Timestamp)
 
 	b.receivedLogBlocks = 0
 	b.receivedLogTx = 0
 	b.receivedLogSTx = 0
```
Comment (davecgh) [inline blocklogger.go]: No need for this anymore since it's not used.

File: blocklogger.go [hunk 6]
```diff
@@ -41,9 +44,11 @@ func (b *blockProgressLogger) logBlockHeight(block *dcrutil.Block) {
 	b.Lock()
 	defer b.Unlock()
 	b.receivedLogBlocks++
-	b.receivedLogTx += int64(len(block.MsgBlock().Transactions))
 	b.receivedLogSTx += int64(len(block.MsgBlock().STransactions))
-
+	b.receivedLogTx += int64(len(block.MsgBlock().Transactions)) + int64(len(block.MsgBlock().STransactions))
```
Comment (davecgh) [inline blocklogger.go]: Exceeds 80.

**Pull request merged**